bot/bot.py

```diff
@@ -52,6 +55,21 @@ async def setup_hook(self) -> None:
 
     async def on_error(self, event: str, *args, **kwargs) -> None:
         """Log errors raised in event listeners rather than printing them to stderr."""
+        e_val = exception()
+
+        if isinstance(e_val, Forbidden):
+            event_to_message_indx = {
+                "on_message": 0,
+                "on_message_edit": 1
+            }
+            message = None
+            if (message_indx_in_args := event_to_message_indx.get(event)) is not None:
+                message = args[message_indx_in_args]
```

Comment:
```suggestion
            message = None
            if event in ("on_message", "on_message_edit"):
                message = args[0] if event == "on_message" else args[1]
```
imo this simpler, and much easier to read.

Reply:
If we allow for this function to handle codes other than 90001, as was my plan, then we need to account for errors that may be raised from other events too though. Say, `on_guild_leave` or w/e. Which would mean this solution won't work (hence my dict and checking if it's `None`).

Reply:
e4b2d5e